spec/views/submissions/show.html.erb_spec.rb

```diff
@@ -13,10 +12,13 @@
 
   it 'renders attributes in <p>' do
     render
-    expect(rendered).to match(/MyText/)
-    expect(rendered).to match(/MyText/)
-    expect(rendered).to match(/MyText/)
-    expect(rendered).to match(/false/)
-    expect(rendered).to match(/false/)
+    
+    assert_select 'form[action=?][method=?]', submission_path(@submission.id), 'post' do
+      assert_select 'textarea[name=?]', 'submission[html]'
+      
+      assert_select 'input[name=?][value=?]', 'submission[base_url]', 'MyText'
+      
+      assert_select 'textarea[name=?]', 'submission[json]'
+    end
```

Comment:
This set of assertions is an improvement over the string matching (which was breaking). This is a quick fix, but I'm sure it could be improved.